assets/less/formstyles.less

```diff
@@ -250,11 +250,9 @@ form.view-query-save {
     border: none;
   }
 
-  input[type="checkbox"]:checked + label {
-    .fonticon-ok;
-  }
   input[type="checkbox"]:checked + label:before {
     font-family: @fauxtonFont;
+    content: "\ea48";
     font-size: 12px;
```

Comment:
This is what I meant about hard coding those problem values into the code to make tehm appear. Instead of `content: "\ea48"` living in the fonticon-ok variable, it's hard coded here. The other option is to restyle the places that use this "checkbox" to use a regular checkbox instead.